cmd/minishift/cmd/daemon/daemon.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package daemon
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var DaemonCmd = &cobra.Command{
+	Use:   "daemon",
+	Short: "Starts a minishift service daemon.",
+	Long:  `Starts a minishift service daemon, this is only to be used internally by Minishift`,
```

Comment:
`Starts a minishift service daemon, this is only to be used internally by Minishift`. This does not not need a comma.